src/Modules/UngradedModules.jl

```diff
@@ -574,6 +618,16 @@ function singular_module(F::FreeMod)
   return Singular.FreeModule(Sx, dim(F))
 end
 
+@doc Markdown.doc"""
+    singular_module(F::FreeMod, ordering::ModuleOrdering)
+
+Create a Singular module from an OSCAR free module over the given Singular polynomial ring.
```

Comment:
For consistency with what you wrote elsewhere:
```suggestion
Create a Singular module from an Oscar free module over the given Singular polynomial ring.
```